decidim-admin/config/locales/en.yml

```
@@ -135,14 +135,26 @@ en:
     admin:
       actions:
         add: Add
+        attachment:
+          new: New %{name}
```

Since we already know it will talk about attachments, I think we can change this to this:

```suggestion
          new: New attachment
```

This change would apply to the rest of the strings, too!